src/main/kotlin/it/pagopa/wallet/controllers/WalletController.kt

```diff
@@ -106,6 +106,28 @@ class WalletController(
         return walletService.findWalletByUserId(xUserId).map { ResponseEntity.ok(it) }
     }
 
+    @SuppressWarnings("kotlin:S6508")
+    override fun notifyWallet(
+        walletId: UUID,
+        orderId: String,
+        walletNotificationRequestDto: Mono<WalletNotificationRequestDto>,
+        exchange: ServerWebExchange
+    ): Mono<ResponseEntity<Void>> {
+        return walletNotificationRequestDto.flatMap { requestDto ->
+            getAuthenticationToken(exchange)
+                .flatMap { securityToken ->
```

Comment:
Add error handling in case of no Authorization header received. The `getAuthenticationToken` returns a Mono.empty in this case so a switchIfEmpty maybe should be add here with a Mono.error being returned, what do you think?

Reply:
Fixed with 60f1613c21a49e7a9e967f931c04fe6a1bc0331d